department-of-veterans-affairs/va.gov-cms
# VACMS-11692: Drupalize create account block

## Description
Adds new "CTA with Links" block type, fields, and entityqueue.

Relates to #11692

## Screenshots
![Screenshot 2023-02-22 at 9 52 02 AM](https://user-images.githubusercontent.com/221539/220713594-dc389e37-53b7-4347-af13-b36789713fb2.png)

![Screenshot 2023-02-22 at 10 59 20 AM](https://user-images.githubusercontent.com/221539/220731755-544c9d84-ab38-4907-a3d6-e99f702c1ac5.png)

## QA steps
As user with Homepage Manager role (you will have to create a new user or edit an existing user and add this role)
1. Navigate to /admin/structure/block/block-content
   - [ ] Validate that there is a block listed with the description: 'Home page create account block' and a block type `CTA with Links`
2. Then edit the block
   - [ ] Validate that the CTA summary text reads: `Create an account to manage your VA benefits and care in one place — anytime, from anywhere.`
   - [ ] Validate that the CTA Primary Button Text reads: `Create account`
   - [ ] Validate that a single Related Info Link(s) is present with url = `/resources/creating-an-account-for-vagov` and text = `Learn how an account helps you`
   - [ ] Validate that the 'Add another link' button appears for up to 3 total Related Info Link(s), and disappears when 3 total link fields are added.
   - [ ] Validate that the current moderation state is set to 'Published'.
3. Navigate to /admin/structure/entityqueue/v2_home_page_create_account/v2_home_page_create_account?destination=/admin/structure/entityqueue
  - [ ] Validate that the subqueue is named: `V2 Home page Create Account`
  - [ ] Validate that it contains a single item: `Home page create account block`

### Definition of Done


### Select Team for PR review
